### ISSUE TITLE
Duplicating packets of withdrawn

### ISSUE THREAD

**Issue by shevchenkodenis** [osrg/gobgp#1050]
on GoBGP 1.10 version 15edc96

These commands will lead to duplicating. 
```
gobgp global rib add 160.0.0.0/24
gobgp global rib del 160.0.0.0/24
```
The issue was also reproduced via native call without gRPC and cli.

Example is shown in attached pcap file.
192.168.5.14 - is a gobgp server
14,15 are duplicated packets
[multiple-withdrawn.pcap.zip](https://github.com/osrg/gobgp/files/409231/multiple-withdrawn.pcap.zip)

**Comment by ishidawataru** [comment]
Are you using the latest version?

The issue should have been fixed by the following commit.

https://github.com/osrg/gobgp/commit/b5e49f1319c37b4da3a010e09fc370d9285bb828

**Comment by shevchenkodenis** [comment]
Oh, it was fixed by b5e49f1319c37b4da3a010e09fc370d9285bb828
